src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/ConnectionServerlessRetryNotification.cs

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.Connection.Contracts
+{
+    /// <summary>
+    /// ConnectionComplete notification mapping entry 
+    /// </summary>
+    public class ConnectionServerlessRetryNotification
```

Comment:
Please revert this for now - this kind of thing doesn't need to be specific to serverless connection retries so if we're going to do it (which I'm not sure we need to) then we should have a separate discussion of how to do it in a more generic manner. 

For the purposes of this PR just fixing the issue by adding the retry is enough - even if that means it just sits there spinning for a bit longer.

Reply:
I was following what Cheena suggested, but I agree it's not necessary.